### PR TITLE
Edited CSS to set tooltip max width

### LINKED ISSUE #5045: Add css to set width of tooltip
Refer #5027 
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [wiki.css](https://github.com/publiclab/plots2/blob/master/app/assets/stylesheets/wiki.css) in the `plots2` repository to add css for the tooltip.
Added css should set the maximun width of the tooltip.
Code of this format can be followed -

`.tooltip-inner {
    max-width: 500px
}`

[See this page](https://publiclab.github.io/community-toolbox/#r=all) for some help in taking your first steps!

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

### PR DESCRIPTION
**Pull request by kvenkman**
Fixes #5045 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by kvenkman** [comment]
Hey @ananyaarun, I can't get the code to run locally because of a ruby versioning error. Is there a way around that?

**Comment by ananyaarun** [comment]
Can you paste a screen shot of the error here ? We can help you out !

**Comment by kvenkman** [comment]
Can't screenshot atm, but the error is along the lines of : 

`Your Ruby version is 2.x.x, but your Gemfile specified 2.4.4`

When I try to `rbenv install 2.4.4` though, rbenv returns an error that it cannot locate version 2.4.4

File: app/assets/stylesheets/wiki.css [hunk 1]
```diff
@@ -75,6 +75,10 @@ a.navbar-offset {
   }
 }
 
+.tooltip-inner { 
+  max-width: 500px
```
Comment (cesswairimu) [inline app/assets/stylesheets/wiki.css]: Hi @kvenkman, Ibelieve we need a semi-colon at the end of the line. Thanks

Comment (cesswairimu) [inline app/assets/stylesheets/wiki.css]: Hi @kvenkman, Ibelieve we need a semi-colon at the end of the line. Thanks

**Comment by govindgoel** [comment]
I am closing this PR as the contributor is inactive since a long period of time
Feel free to reopen it if needed.

**Pull request closed without merging**